runtimes/datascience/ubi9-python-3.11/utils/requirements-elyra.txt

```diff
@@ -1,24 +1,24 @@
 # Copied from: https://github.com/elyra-ai/elyra/blob/main/etc/generic/requirements-elyra.txt
 
 # This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
-ipykernel==6.13.0
-ipython==8.10.0
+ipykernel==6.29.5
```

Comment:
@harshad16 I've manually updated this file to the latest versions, and I've applied the same versions to the [Pipfiles](https://github.com/opendatahub-io/notebooks/blob/391b967d02eda0468f3cfc5e7bc0994d863a784a/runtimes/datascience/ubi9-python-3.11/Pipfile) example as well. 

The packages on the requirments.txt on elyra repo looks old: https://github.com/elyra-ai/elyra/blob/main/etc/generic/requirements-elyra.txt 

Since I wasn't certain which versions to use, I opted to go with the latest updates.

Reply:
Personally I would remove any changes to these `requirements-elyra.txt` files from this PR. Or delete them completely to comply with: https://issues.redhat.com/browse/RHOAIENG-11068.

Reply:
All the changes look good.
as we are not using the requirements-elyra.txt,  we would anyway use the https://issues.redhat.com/browse/RHOAIENG-11068, to de-clutter this.

Reply:
Tnx for pointing this out, the changes have been incorporated under this commit https://github.com/opendatahub-io/notebooks/pull/706/commits/664ea3767b630837605daa3db82e72c718ca4bca